src/Microsoft.AspNet.Server.Kestrel/Infrastructure/MemoryPoolIterator2.cs

```diff
@@ -439,7 +441,35 @@ public int Seek(Vector<byte> byte0Vector, Vector<byte> byte1Vector, Vector<byte>
             }
         }
 
-        private static int FindFirstEqualByte(Vector<byte> byteEquals)
+        /// <summary>
+        /// Find 
+        /// </summary>
+        /// <param name="byteEquals"></param>
+        /// <returns>The first index of the result vector if found otherwise return int.MaxValue</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Think this is the first time we've used this attribute in Kestrel. It's probably better to profile and add these attributes all at once if/when we decide to do that.